### PR TITLE
Fix worker connect

### PR DESCRIPTION
**Pull request by nicoburniske**

### CODE REVIEW

File: src/clients/worker.rs [hunk 1]
```diff
@@ -345,12 +346,19 @@ impl WorkerClient for WorkerClientLive {
 
             match msg {
                 None => {}
-                Some(msg) => match msg {
-                    InstanceConnectMessage::Message { message } => {
+                Some(msg) => match msg.event {
+                    WorkerEvent::Stdout(StdOutLog { message }) => {
                         print!("{message}")
                     }
-                    InstanceConnectMessage::Error { error } => {
```
Comment (senia-psm) [inline src/clients/worker.rs:352]: Do we have a way to report `golem` internal errors?

File: src/clients/worker.rs [hunk 2]
```diff
@@ -345,12 +346,19 @@ impl WorkerClient for WorkerClientLive {
...
-                        eprintln!("Connection error: {error}")
+                    WorkerEvent::Stderr(StdErrLog { message }) => {
+                        eprint!("{message}")
+                    }
+                    WorkerEvent::Log(Log {
+                        level,
+                        context,
+                        message,
+                    }) => {
+                        println!("[{level}] [{context}] {message}")
```
Comment (senia-psm) [inline src/clients/worker.rs:361]: It should be `stderr`.

**Comment by senia-psm** [comment]
`golem-cli` `stdout` is for result. And `worker` `stderr` messages is a kind of `golem-cli` result.

`golem-cli` `stderr` is for logs (including internal error reporting).

File: src/clients/worker.rs (hunk 2 above)
Comment (senia-psm) [reply src/clients/worker.rs:361]: Consider using `tracing::event!(map_level(level), message, context=context)`

**Comment by nicoburniske** [comment]
I'm just trying to translate scala code in

```scala
  .mapZIO { // TODO: send structured (json) data on the web-socket not just the string messages
    case Event.Empty          =>
      ZIO.unit
    case Event.Stdout(stdout) =>
      ZIO.logDebug(s"Forwarding worker standard output: ${stdout.message}") *>
        channel.send(messageChannelEvent(stdout.message))
    case Event.Stderr(stderr) =>
      ZIO.logDebug(s"Forwarding worker standard error: ${stderr.message}") *>
        channel.send(messageChannelEvent(stderr.message))
    case Event.Log(log)       =>
      val asString = s"[${log.level}] [${log.context}] ${log.message}"
      ZIO.logDebug(s"Forwarding worker log: $asString") *>
        channel.send(messageChannelEvent(asString))
  }
```

**Review by senia-psm — APPROVED** [review]

**Pull request merged**